### PR TITLE
test: add test cases

### PR DESCRIPTION
**Pull request by dead-horse**
This change is [Reviewable](https://reviewable.io/reviews/eggjs/egg-cluster/3)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,25 +27,32 @@
...
   "devDependencies": {
     "autod": "^2.6.1",
-    "egg-bin": "^1.0.1",
+    "coffee": "^3.2.4",
+    "egg": "*",
+    "egg-bin": "^1.0.2",
     "egg-ci": "^1.0.2",
-    "eslint": "^3.0.1",
-    "eslint-config-egg": "^3.0.1"
+    "egg-mock ": "*",
```
Comment (popomore) [inline package.json]: 多了个空格，ci 挂了

**Comment by popomore** [comment]
用例还是挂的，要我来看么

**Comment by dead-horse** [comment]
看起来是 egg 升级了导致的，我明天看一下修复吧

**Comment by dead-horse** [comment]
测试用例好了

**Comment by fengmk2** [comment]
+1

**Pull request merged**